.github/workflows/test.yml

```diff
@@ -13,5 +13,12 @@ jobs:
     - uses: actions/checkout@v2
       with:
         fetch-depth: 0
-    - uses: actions/setup-dotnet@v1
-    - run: dotnet test -c Release
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+    - name: Dotnet Test
+      run: dotnet test -c Release /p:CollectCoverage=true /p:CoverletOutputFormat=cobertura /p:CoverletOutput=TestResults/ 
```

Comment:
What is the reason of changing from `opencover`  to `cobertura`?

Reply:
The test hung with opencove, though not sure if it's caused by format. Lemme try again.